### ISSUE TITLE
Struggling with setting `risk_free_return_rate` in `backtest.json`

### ISSUE THREAD

**Issue by sjhermanek** [Blankly-Finance/Blankly#90]
## Description
I'm struggling with setting `risk_free_return_rate` and see it propagated when printing results from a backtest. Could you please help?

## backtest.json
```
{
  "price_data": {
...
  },
  "settings": {
...
    "risk_free_return_rate": 0.01
  }
}
```

## Output when print(results)
```
Blankly Metrics: 
Compound Annual Growth Rate (%):   661.0%
Cumulative Returns (%):            64.0%
Max Drawdown (%):                  8.0%
Variance (%):                      6.36%
Sortino Ratio:                     4.92
Sharpe Ratio:                      3.68
Calmar Ratio:                      19.29
Volatility:                        0.03
Value-at-Risk:                     654.94
Conditional Value-at-Risk:         24.68
Risk Free Return Rate:             0.0
Resampled Time:                    86400.0
```

Tested on last main branch installed using pip.

**Comment by sjhermanek** [comment]
Also weird flex about that CAGR but ok :D

**Comment by EmersonDove** [comment]
Interesting - I could have broken it a few days ago, I was messing around with the way these exports are created. 

This is unrelated but we were actually discussing last night that some of the calculations for these metrics might not be correct which is something we're going to look into.

I'm going to run it now and make sure its propagating.

**Comment by EmersonDove** [comment]
Interesting I just ran it and got the correct result:
```
Blankly Metrics: 
...
Conditional Value-at-Risk:         56.1
Risk Free Return Rate:             0.1
...
```

Can you try running `pip install --upgrade blankly` - I've released a few versions since 1.14 straight from the dev branch.

**Comment by sjhermanek** [comment]
`pip install --upgrade blankly`` did it, thanks!
